src/components/HathorHeader.js

```diff
@@ -61,7 +59,7 @@ const HathorHeader = (props) => {
 
   let extraStyle = {};
   if (props.withBorder) {
-    extraStyle = { borderBottomWidth: STYLE.borderWidth };
+    extraStyle = { borderBottomWidth: 1 };
```

Comment:
Why is it hardcoded as 1 and not using the style?

Reply:
Actually the style was removed, as on 326 it was resolved that we would no longer have different border widths for Android and iOS.

Adding the many variations of each component's borders would not improve maintainability of the code, so I found it best to just let these border widths be hardcoded again.